packages/stream_chat_flutter/lib/src/attachment/video_attachment.dart

```diff
@@ -11,16 +11,18 @@ class VideoAttachment extends AttachmentWidget {
   final MessageTheme messageTheme;
   final ShowMessageCallback onShowMessage;
   final ValueChanged<ReturnActionType> onReturnAction;
+  final VoidCallback onAttachmentTap;
 
-  const VideoAttachment({
-    Key key,
-    @required Message message,
-    @required Attachment attachment,
-    Size size,
-    this.messageTheme,
-    this.onShowMessage,
-    this.onReturnAction,
-  }) : super(key: key, message: message, attachment: attachment, size: size);
+  const VideoAttachment(
+      {Key key,
+      @required Message message,
+      @required Attachment attachment,
+      Size size,
+      this.messageTheme,
+      this.onShowMessage,
+      this.onReturnAction,
+      this.onAttachmentTap})
```

Comment:
can you add a trailing comma?
https://flutter.dev/docs/development/tools/formatting#using-trailing-commas

Reply:
done